openapi/generator.go

```diff
@@ -1161,3 +1177,21 @@ func fieldNameFromTag(sf reflect.StructField, tagName string) string {
 	}
 	return name
 }
+
+/// parseExampleValue is used to transform the string representation of the example value to the correct type.
+func parseExampleValue(rType reflect.Type, value string) (interface{}, error) {
```

Comment:
Not a huge deal, but for consistency accros the library, please rename the param to `typ` or `t`.